packages/authenticator/src/js/AuthStore/AuthStore.js

```diff
@@ -0,0 +1,9 @@
+/*
```

Comment:
when requiring (and importing) files, `AuthStore`, `AuthStore.js` and `authstore` aren't recognized as the same file, so when importing a singleton this way, you'll have different instances. this file is more or less an experiment if proxying will help (it does). its not needed because we do the same thing in `index.js`